### DISCUSSION TITLE
About verification code

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38637] [Other Feature Feedback, Questions, & Ideas]
Where can i find 25 digit verification code in git hub for azure free subscription academic verification... Can you please tell me

**Comment by DSDmark** [comment]
Hi @shaiksadaf,

you can go with email verification, either.

Here is step to verification:- https://blog.glugmvit.com/activate-azure/

or if you're not understand, just simply go there and fill all details they ask:-
https://azure.microsoft.com/en-us/free/students/?WT.mc_id=academic-9938-cxa
